yamui.c

```diff
@@ -1438,7 +1439,7 @@ static const char opt_short[] = "a:i:p:s:t:hxnd";
 int
 main(int argc, char *argv[])
 {
-	bool debugging = false;
```

Comment:
Given the nature of problems it was causing elsewhere I'd make this default to: do_cleanup = true

and then flip it to false with some option, say "--no-cleanup" or "--skip-cleanup" 
(the "--debug" was hidden/undocumented thing and can be dropped altogether)

Reply:
Added `--skip-cleanup`.